test/unit/storage/adapters/pgp_adapter.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable mocha/no-pending-tests */
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const rewire = require('rewire');
+const Promise = require('bluebird');
+const BaseAdapter = require('../../../../storage/adapters/base_adapter');
+
+const PgpAdapter = rewire('../../../../storage/adapters/pgp_adapter');
+
+const loggerStub = {
+	info: sinonSandbox.stub(),
+	log: sinonSandbox.stub(),
+};
+
+const validOptions = {
+	inTest: true,
+	sqlDirectory: '/sql/directory',
+	logger: loggerStub,
+};
+
+describe('PgpAdapter', () => {
+	afterEach(() => {
```

Comment:
maybe we can use the same pattern from `elements` we have set global_hook for `sandbox.restore`

Reply:
This would require refactoring all the old tests which is out of the scope of this PR imo.